regression/tests/studio/test_studio_home.py

```diff
@@ -0,0 +1,49 @@
+import os
+from bok_choy.web_app_test import WebAppTest
+from regression.pages.studio.login_studio import StudioLogin
+from regression.pages.studio.studio_home import DashboardPageExtended
+from edxapp_pages.pages.studio.overview import CourseOutlinePage
+from regression.pages.studio import BASE_URL
+from bok_choy.page_object import XSS_INJECTION
+
+
+class StudioHomeTest(WebAppTest):
+    """
+    Test for navigating to the Studio Home page
+    """
+
+    DEMO_COURSE_USER = os.environ.get('USER_LOGIN_EMAIL')
+    DEMO_COURSE_PASSWORD = os.environ.get('USER_LOGIN_PASSWORD')
+
+    def setUp(self):
+        """
+        Initialize the page object
+        """
+        super(StudioHomeTest, self).setUp()
+        self.studio_login_page = StudioLogin(self.browser)
+        self.studio_login_page.url = BASE_URL + '/signin'
+        self.studio_login_page.visit()
+        self.studio_login_page.login(self.DEMO_COURSE_USER, self.DEMO_COURSE_PASSWORD)
+
+    def test_studio_course_select(self):
+        """
+        Verifies that user can select a course and navigate to its course outline page
+        """
+        self.studio_home_page = DashboardPageExtended(self.browser)
+
+        self.course_info = {
+            'org': 'test_org',
+            'number': self.unique_id,
+            'run': 'test_run',
+            'display_name': 'Test Course' + XSS_INJECTION + self.unique_id
```

Comment:
Yes. The only issue was browser authentication which was handled already. So we can use this now.